wpilibc/src/generate/main/native/cpp/hid.cpp.jinja

```diff
@@ -90,3 +90,16 @@ bool {{ ConsoleName }}Controller::GetTouchpadReleased() {
   return GetRawButtonReleased(Button::kTouchpad);
 }
 {% endif %}
+
+void {{ ConsoleName }}Controller::InitSendable(const wpi::SendableBuilder& builder) {
+  builder.SetSmartDashboardType("HID");
+  {% for trigger in triggers %}
+  builder.AddDoubleProperty("{{ capitalize_first(trigger.name) }}", Get{{ capitalize_first(trigger.name) }}, nullptr);
```

Comment:
iirc method refs like these don't work properly in C++ due to capture semantics etc; a lambda with an explicit capture of this is what we typically do: https://github.com/wpilibsuite/allwpilib/blob/main/wpilibc%2Fsrc%2Fmain%2Fnative%2Fcpp%2FAnalogPotentiometer.cpp#L48